src/helpers/chunked-data.test.ts

```diff
@@ -0,0 +1,87 @@
+import { ChunkedData, ChunkType } from "./";
+
+describe("test chunked data", () => {
+  const exampleData = {
+    key3: 3,
+    key1: 1,
+    key2: 2,
+  };
+
+  it("Should create a chunked data and get valid total chunk", async () => {
+    expect(new ChunkedData<number>(exampleData, 1).totalChunks).toBe(3);
+    expect(new ChunkedData<number>(exampleData, 2).totalChunks).toBe(2);
+  });
+
+  it("Should create a chunked data and get keys sorted", async () => {
+    expect(new ChunkedData<number>(exampleData, 2).keys).toEqual([
```

Comment:
I didn't understand well this test because "If the chunk size is 2, why keys has 3 elements?" But i understood after...

Reply:
So it is ok now ?

Reply:
yes